cg/cli/clean.py

```diff
@@ -127,6 +188,103 @@ def scout_finished_cases(
         context.invoke(hk_alignment_files, bundle=bundle, yes=yes, dry_run=dry_run)
 
 
+@clean.command("hk-case-bundle-files")
+@click.option(
+    "--days-old",
+    type=int,
+    default=365,
+    help="Clean all files with analysis dates older then given number of days",
+)
+@click.option("-d", "--dry-run", is_flag=True, help="Shows cases and files that would be cleaned")
+@click.pass_context
+def hk_case_bundle_files(context: CGConfig, days_old: int, dry_run: bool = False):
+    """Clean up all files for all pipelines but those whitelisted"""
+    status_db: Store = context.obj.status_db
+    housekeeper_api: HousekeeperAPI = context.obj.housekeeper_api
+    before: datetime = datetime.now() - timedelta(days=days_old)
+
+    size_cleaned = 0
+    pipeline: Pipeline
+    for pipeline in Pipeline:
+
+        protected_tags_lists = PIPELINE_PROTECTED_TAGS.get(pipeline)
+        if protected_tags_lists:
+            LOG.debug(f"protected tags defined for {pipeline} {protected_tags_lists}")
+        else:
+            LOG.debug(f"no protected tags defined for {pipeline}, skipping")
+            continue
+
+        analyses: Query = status_db.get_analyses_before_date(pipeline=pipeline, before=before)
+
+        analysis: models.Analysis
+        for analysis in analyses:
+            LOG.info(f"Cleaning analysis {analysis}")
+            bundle_name: str = analysis.family.internal_id
+            hk_bundle_version: Optional[hk_models.Version] = housekeeper_api.version(
+                bundle=bundle_name, date=analysis.started_at
+            )
+            if not hk_bundle_version:
+                LOG.warning(
+                    f"Version not found for "
+                    f"bundle:{bundle_name}; "
+                    f"pipeline: {analysis.pipeline}; "
+                    f"date {analysis.started_at}"
+                )
+                continue
+
+            LOG.info(
+                f"Version found for "
+                f"bundle:{bundle_name}; "
+                f"pipeline: {analysis.pipeline}; "
+                f"date {analysis.started_at}"
+            )
+            version_files: List[hk_models.File] = housekeeper_api.get_files(
+                bundle=analysis.family.internal_id, version=hk_bundle_version.id
+            ).all()
+
+            version_file: hk_models.File
+            for version_file in version_files:
+                LOG.info(f"File {version_file.full_path} has the tags {version_file.tags}.")
+                version_file_tags: [str] = [tag.name for tag in version_file.tags]
+
+                has_protected_tags: bool = False
+                for protected_tags in protected_tags_lists:
```

Comment:
I think it might confuse to have different wording on left and right side in a `for item in items` so I kind of prefer `for protected_tags in protected_tags_lists` to `for file_protected_tags in pipeline_protected_tags`